Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -209,10 +213,10 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
             return
         }
 
-        await rekey()
+        try? await rekey()
     }
 
-    private func rekey() async {
+    private func rekey() async throws {
```

Comment:
Let's rethrow so we can detect wake attempt failures due to these errors.

Reply:
Perfect :+1: